Xamarin.Forms.Platform.UAP/BoxViewRenderer.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using System.ComponentModel;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Shapes;
 
 namespace Xamarin.Forms.Platform.UWP
 {
-	public class BoxViewRenderer : ViewRenderer<BoxView, Windows.UI.Xaml.Shapes.Rectangle>
+	public class BoxViewRenderer : ViewRenderer<BoxView, Border>
```

Comment:
breaking change

Reply:
@samhouts  I wondered if that would be case.  I will do a little more investigation into how to maybe do it with a rectangle shape, but I"m not optimistic on that.

In the mean time, I'll create a new renderer, any suggestions on how you would name that?  BoxViewRenderer2?